.github/workflows/publish.yaml

```diff
@@ -5,13 +5,16 @@ name: Publish
 on:
   pull_request:
     branches: [ master ]
+    types: [opened, synchronize, reopened, labeled, unlabeled]
   push:
     tags: [ '[A-z]+-v[0-9]+.[0-9]+.[0-9]+*' ]
 
 jobs:
   publish:
     if: ${{ github.repository_owner == 'dart-lang' }}
     uses: dart-lang/ecosystem/.github/workflows/publish.yaml@main
+    with:
+      write-comments: false
```

Comment:
Don't we WANT comments?

Reply:
Hmmmm the build depot has a separate publish comments yaml, so maybe here it does need to write comments. Let's try with just the change above and see what happens :) done.